bool is_stop
image bounding_box